sdk/textanalytics/ai-text-analytics/samples/typescript/README.md

```diff
@@ -73,6 +74,7 @@ Take a look at our [API Documentation][apiref] for more information about the AP
 
 [alternativedocumentinput]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/textanalytics/ai-text-analytics/samples/typescript/src/alternativeDocumentInput.ts
 [analyzesentiment]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/textanalytics/ai-text-analytics/samples/typescript/src/analyzeSentiment.ts
+[analyzesentimentwithopinionmining]: https://github.com/deyaaeldeen/azure-sdk-for-js/blob/opinion-mining/sdk/textanalytics/ai-text-analytics/samples/typescript/src/analyzeSentimentWithOpinionMining.ts
```

Comment:
same comment here

Reply:
[Done](https://github.com/Azure/azure-sdk-for-js/issues/10559).